docs/api.md

```diff
@@ -115,6 +115,15 @@ like `st.line_chart` and `st.area_chart`.
 .. autofunction:: streamlit.deck_gl_chart
 .. autofunction:: streamlit.graphviz_chart
 .. autofunction:: streamlit.map
+```
+
+## Display media
+
+
+
```

Comment:
Why so many newlines?

Reply:
This is a bug. Let me take a look and figure out what happened.

Reply:
Pushing a fix now.